Earthfile

```diff
@@ -48,4 +48,4 @@ repo-config:
 edit-docs:
     LOCALLY
 
-    RUN ./earthly/docs/dev/local.py cat-ci-docs:latest
\ No newline at end of file
+    DO ./earthly/docs/dev/local.py cat-ci-docs:latest
```

Comment:
This should be `RUN` , `DO` is for calling function

Reply:
gotcha!